MyOwnORM/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,203 @@
+                        PersonId = 1
+                    },
+                    new Book()
+                    {
+                        Id = 2,
+                        Title = "Title",
+                        PersonId = 1
+                    },
+                    new Book()
+                    {
+                        Id = 3,
+                        Title = "Title",
+                        PersonId = 1
+                    }
+                }
+            };
+            //Cascade insert
+            await context.People.InsertCascadeAsync(newPerson1);
+            var updatePerson = new Person()
+            {
+                Id = 1,
+                Name = "Nikita Beloshapka",
+                Age = 20
+            };
+            await context.People.UpdateAsync(newPerson);
+            var updatePerson1 = new Person()
+            {
+                Id = 1,
+                Name = "Nikita Beloshapka",
+                Age = 20,
+                Author = new Author
+                {
+                    Id = 1,
+                    Name = "Danya Zbitnyev",
+                    PersonId = 1,
+                },
+                Sportsmen = new Sportsmen
+                {
+                    Id = 1,
+                    Name = "Kiriil Drygach",
+                    IdPerson = 1,
+                },
+                Book = new List<Book>
+                {
+                    new Book()
+                    {
+                        Id = 1,
+                        Title = "Title",
+                        PersonId = 1
+                    },
+                    new Book()
+                    {
+                        Id = 2,
+                        Title = "Title",
+                        PersonId = 1
+                    },
+                    new Book()
+                    {
+                        Id = 3,
+                        Title = "Title",
+                        PersonId = 1
+                    }
+                }
+            };
+            //Cascade update
+            await context.People.UpdateCascadeAsync(newPerson1);
+            await context.People.DeleteAsync(p => p.Id == 1);
+            //Cascade delete
+            await context.People.DeleteCascadeAsync(p => p.Id == 1);
+            dynamic p3 = await context.People.FromSqlRawAsync("SELECT * FROM Person WHERE Id = 2");
+            //Eager loading
+            var p4 = context.People.IncludeAsync(p => p.Author);
+            Expression<Func<Person, object>>[] expressions = new Expression<Func<Person, object>>[]
+            {
+                a => a.Author,
+                b => b.Book,
+                s => s.Sportsmen
+            };
+            //Eager loading with expression arrays
+            var p5 = await context.People.IncludeAsync(expressions);
+        }
+        public static void Main()
+        {
+            StartProgram();
+        }
+    }
+    [Table("Person")]
+    public class Person
```

Comment:
Why don't you use ready domain model that I gave you? Even if you created own, add to domain models inheritance,  and apply all attributes at leats 2 times.

Reply:
Fixed